examples/gno.land/r/gnoland/faucet/faucet_test.gno

```diff
@@ -54,14 +61,21 @@ func TestPackage(t *testing.T) {
 	std.TestSetOrigCaller(controlleraddr1)
 	assertErr(t, faucet.Transfer(test1addr, 1000000))
 
-	// add more than 3 controllers
-
+	// duplicate controller
 	std.TestSetOrigCaller(adminaddr)
 	assertNoErr(t, faucet.AdminAddController(controlleraddr1))
 	assertErr(t, faucet.AdminAddController(controlleraddr1))
+	// add more than more than allowed controllers
 	assertNoErr(t, faucet.AdminAddController(controlleraddr2))
 	assertNoErr(t, faucet.AdminAddController(controlleraddr3))
-	assertErr(t, faucet.AdminAddController(controlleraddr4))
+	assertNoErr(t, faucet.AdminAddController(controlleraddr4))
+	assertNoErr(t, faucet.AdminAddController(controlleraddr5))
+	assertNoErr(t, faucet.AdminAddController(controlleraddr6))
+	assertNoErr(t, faucet.AdminAddController(controlleraddr7))
+	assertNoErr(t, faucet.AdminAddController(controlleraddr8))
+	assertNoErr(t, faucet.AdminAddController(controlleraddr9))
+	assertNoErr(t, faucet.AdminAddController(controlleraddr10))
+	assertErr(t, faucet.AdminAddController(controlleraddr11))
```

Comment:
You can simplify this by running it in a loop (the checks that an address is indeed an admin controller), and run a simple assert that other addresses are not.

For example, if you utilize the helper method mentioned in one of my previous comments, you can write this check like this:

```go
// Assuming you have a slice of controllers previously defined
var controllers []crypto.Address = ...

// Assuming the last controller cannot be added
for _, controller := range controllers[:len(controllers)-1] {
    assert.NoErr(t, faucet.AdminAddController(controller))
}

// Make sure the last controller cannot be added
assertErr(t, faucet.AdminAddController(controllers[len(controllers)-1]))
```

You can simplify this even more, but I think you get the example I'm trying to explain

Reply:
Please see my above comment for this testing case.